hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -826,7 +826,8 @@ else if ( batchSize > 1 ) {
 				colAliases[k] = thing.getAlias( dialect, prop.getValue().getTable() );
 				if ( thing.isFormula() ) {
 					foundFormula = true;
-					( (Formula) thing ).setFormula( substituteBrackets( ( (Formula) thing ).getFormula() ) );
+					// ( (Formula) thing ).setFormula( substituteBrackets( ( (Formula) thing ).getFormula() ) );
+					// TOOD: uncomment the above statement when this#substituteBrackets(String) is implemented
```

Comment:
Do you have a card for this so that we don't forget about it?

Reply:
Nope. In current v6 development, it would be ideal to fix issues related as much as possible. I left a `//TODO` comment above so I think it might be enough. There are too many `NotYetImplementedFor6Exception()` statements here and there in current v6 branch and creating a specific card for each of them might be too fine-grained, IMHO.
However, I did fix the typo (//TOOD -> //TODO).